20_1 button: add getting started (#1773)
DevExpress/devextreme-documentation: concepts/05 UI Components/Button/00 Getting Started with Button/00 Getting Started with Button.md

> This tutorial shows how to add a Button to a page, apply styling, and configure its core features. As a result, you will create the following UI component:

Reviewer: ```suggestion
This tutorial shows how to add a Button to a page, apply styling, and configure its core features.
```

> The Button is the UI component that performs an action when a user clicks it. This tutorial shows how to add the Button to a page, apply styling, and configure the core features. As a result, you will get the following UI component:

Reviewer 2: I wouldn't start with the button definition. You can remove it and start with the second sentence. It's just as clear. 

In general, I'd replace the word "get" with another verb, if I can. Books, such as "Misplaced Modifier" refer to "get" as a "weak and boring" verb. 

 ```suggestion
This tutorial shows how to add a Button to a page, apply styling, and configure its core features. As a result, you will create the following UI component:
```

<div class="simulator-desktop-container" data-view="/Content/Applications/20_1/GettingStartedWith/Button/index.html, /Content/Applications/20_1/GettingStartedWith/Button/index.js, /Content/Applications/20_1/GettingStartedWith/Button/index.css"></div>

> Refer to the following sections for details on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-button" target="_blank">getting-started-with-button</a>.

Reviewer: ```suggestion
Refer to the following sections for information on each configuration step. The full code is available in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-button" target="_blank">getting-started-with-button</a>.
```
